org.jacoco.core.test/src-java8/org/jacoco/core/test/validation/targets/BadCycleInterface.java

```diff
@@ -16,7 +16,7 @@
 	public interface Base {
 		static final Object BASE_CONST = new Child() {
 			{
-				Stubs.nop("base clinit"); // $line-1$
+				Stubs.nop("base clinit"); // $line-baseclinit$
```

Comment:
@marchof I think that full execution order here has an interest only as a demonstration. What really matters here and in `BadCycleClass` - is that some method of a child can be executed before its `clinit` method. And we test exactly this - such situation is properly handled and we do not rely on false assumption that `clinit` executed before everything else. So indeed it seems reasonable to have an assertion about this, but assertion for full sequence is probably overkill.